typings/mysql/lib/Connection.d.ts

```diff
@@ -431,4 +431,7 @@ declare class Connection extends QueryableBase(ExecutableBase(EventEmitter)) {
   sequenceId: number;
 }
 
-export { Connection };
+declare class Connection extends BaseConnection {
```

Comment:
Changed this to fix the pool connection inheritance giving a type error on reimplementing the promise